demo/src/app/examples/advanced/datatable-integration/datatable.type.ts

```diff
@@ -0,0 +1,92 @@
+import { Component, OnInit, Input, ViewChild, TemplateRef } from '@angular/core';
+import { FormlyFieldConfig, FieldArrayType, FormlyFormBuilder } from '@ngx-formly/core';
+import { TableColumn } from '@swimlane/ngx-datatable/release/types';
+
+
+@Component({
+  selector: 'app-table-type',
+  template: `
+<ngx-datatable
+  #table  class="bootstrap" 
+  [rows]="model"
+  [columns]="columns"
+  [columnMode]="columnMode"
+  [rowHeight]="rowHeight"   
+  [headerHeight]="headerHeight"      
+  [footerHeight]="footerHeight"
+  [limit]="limit"  
+  [scrollbarH]="scrollbarH"      
+  [reorderable]="reorderable"    
+  [externalSorting]="true"
+  [selected]="selected"
+  [selectionType]="'single'"
+  (sort)="onSort($event)"
+  (select)='onSelect($event)'>     
+  
+  <ng-template #genericcolumn ngx-datatable-cell-template let-rowIndex="rowIndex" let-value="value" let-row="row" let-column="column">
+    <formly-field             
```

Comment:
use `formly-group` instead

Reply:
I changed `getFields` in `getField` and continue to use `formly-field`

Reply:
The ng-template is a template for each column and used for each cell. Each cell match with the single field in this sense I think is correct to use formly-field or no?

Reply:
@aitboudad you have any suggestion?

Reply:
it's ok then!